app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt

```diff
@@ -72,6 +74,43 @@ fun WatchScreen(
     }
 }
 
+@Composable
+fun EditMemoScreen(
+    logId: Int,
+    memo: String,
+    onButtonClicked: (Int, String) -> Unit = { _, _ -> }
+) {
+    val (text, setText) = remember { mutableStateOf(memo) }
+    EditMemoScreen(
+        memo = text,
+        onMemoEdited = setText,
+        onButtonClicked = { onButtonClicked(logId, text) }
```

Comment:
```suggestion
        onButtonClicked = { onButtonClicked(logId, text) }
    )
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>